src/app/components/client/toolbar/UserMenu.module.scss

```diff
@@ -0,0 +1,94 @@
+@import "../../../tokens";
+
+$button-size: 42px;
+
+.trigger {
+  background: white;
+  border-radius: 50%;
+  border-style: none;
+  cursor: pointer;
+  height: $button-size;
+  padding: 0;
+  width: $button-size;
+
+  @media screen and (max-width: $screen-md) {
+    margin-right: $spacing-sm;
+  }
+
+  &:focus,
+  &:hover {
+    outline: 2px solid $color-purple-70;
+  }
+}
+
+.menuList {
+  border-radius: $border-radius-md;
+  box-shadow: $box-shadow-md;
+  font: $text-body-md;
+  list-style-type: none;
+  min-width: $content-xs;
+  outline: 0;
+  padding: $spacing-sm 0;
+  position: relative;
+  right: 0;
+
+  &::before {
+    $tooltip-size: 12px;
+    background-color: white;
+    content: "";
+    height: $tooltip-size;
+    position: absolute;
+    right: calc($spacing-md + $button-size * 0.5);
+    top: 0;
+    transform: translateY(-50%) rotate(45deg);
+    width: $tooltip-size;
+
+    @media screen and (max-width: $screen-md) {
+      right: calc($button-size * 0.5);
+    }
+  }
+}
+
+.menuItemWrapper {
+  cursor: pointer;
+  padding: $spacing-sm calc($spacing-md * 1.5);
```

Comment:
I appreciate that we're using tokens, but if we're just reverse engineering the number we're looking for with a calculation based on an existing token, we might as well be using the number directly - it's not like it's going to be a more consistent rhythm, or scale correctly if we ever decide to change those numbers :sweat_smile: 

Any reason a `$spacing-lg` doesn't suffice here? Isn't it the exact same size?

Reply:
Right, this is matching `$spacing-lg`. I’ll try sticking to the default tokens for tokens.